scripts/link-mirabox.mjs

```
@@ -0,0 +1,67 @@
+loadEnvLocal();
+
+const dest = process.env.MIRABOX_PLUGINS_DIR;
+if (!dest) {
+  console.error("Error: MIRABOX_PLUGINS_DIR is not set.");
+  console.error("Set it in your shell or in .env.local at the repo root, e.g.:");
+  console.error('  MIRABOX_PLUGINS_DIR="C:\\\\Users\\\\you\\\\AppData\\\\Roaming\\\\VSD Craft\\\\Plugins"');
+  process.exit(1);
+}
+
+if (!existsSync(SOURCE)) {
+  console.error(`Error: Mirabox plugin folder does not exist. Run \`pnpm build\` first.\n  ${SOURCE}`);
```

ESLint

[error] 'console' is not defined.

(no-undef)